model-optimizer/mo/graph/graph.py

```diff
@@ -1097,6 +1101,27 @@ def rename_nodes(nodes: List[tuple]):
     for node, name in nodes:
         rename_node(node, name)
 
+
+def get_attribute_between_nodes(node1: Node, node2: Node, attr_name: str):
+    for edge_idx in node1.out_edges():
```

Comment:
Please, add a comment describing the function logic and why it was introduced.

Reply:
And mention that the function should be used during the front phase only.

Reply:
And if there are multiple edges between two nodes then it returns an attribute from any of these nodes.